doc/api/stream.md

```diff
@@ -1781,6 +1845,57 @@ const myWritable = new Writable({
 });
 ```
 
+#### writable.\_construct(options, callback)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object} Options passed to constructor.
+* `callback` {Function} Call this function (optionally with an error
+  argument) when finished writing any remaining data.
```

Comment:
The description here for `callback` looks like it's copied from somewhere else? It should be called when the initialization has finished, not when no more data has been written. Similarly for `readable._construct()`'s description.